new-integration-tests/src/test/java/oracle/weblogic/kubernetes/assertions/impl/Kubernetes.java

```diff
@@ -429,6 +433,74 @@ public static V1Service getService(
     return null;
   }
 
+  /**
+   * Get V1Deployment object for the given  name, label and namespace.
+   * @param deploymentName name of the deployment to look for
+   * @param label the key value pair with which the deployment is decorated with
+   * @param namespace the namespace in which to check for the deployment
+   * @return V1Deployment object if found otherwise null
+   * @throws ApiException when there is error in querying the cluster
+   */
+  public static V1Deployment getDeployment(
+          String deploymentName, Map<String, String> label, String namespace)
+          throws ApiException {
+    String labelSelector = null;
+    if (label != null) {
+      String key = label.keySet().iterator().next().toString();
+      String value = label.get(key).toString();
+      labelSelector = String.format("%s in (%s)", key, value);
+      logger.info(labelSelector);
+    }
+    V1DeploymentList v1DeploymentList = listDeployments(namespace);
+
+    for (V1Deployment deployment : v1DeploymentList.getItems()) {
+      if (deployment.getMetadata().getName().equals(deploymentName.trim())
+              && deployment.getMetadata().getNamespace().equals(namespace.trim())) {
+        logger.info("Deployment Name : " + deployment.getMetadata().getName());
+        logger.info("Deployment Namespace : " + deployment.getMetadata().getNamespace());
+        logger.info("Deployment status : " + deployment.getStatus().toString());
+        Map<String, String> labels = deployment.getMetadata().getLabels();
+        if (labels != null) {
+          for (Map.Entry<String, String> entry : labels.entrySet()) {
+            logger.log(Level.INFO, "Label Key: {0} Label Value: {1}",
+                    new Object[]{entry.getKey(), entry.getValue()});
+          }
+        }
+        return deployment;
+      }
+    }
+    return null;
+  }
+
+  /**
+   * Checks if an deployment  is running in a given namespace.
```

Comment:
extra blank space between the word  "deployment"  and "is"

Reply:
done